src/test/java/com/hocs/test/glue/HomepageStepDefs.java

```diff
@@ -1,12 +1,20 @@
 package com.hocs.test.glue;
 
 import com.hocs.test.pages.homepage.Homepage;
+import cucumber.api.java.en.And;
 import cucumber.api.java.en.Then;
 
 public class HomepageStepDefs {
 
     private Homepage homepage;
 
+
+    @And("^I am returned to my home screen$")
+    public void returnedToHomeScreen() {
+        homepage.assertCreateSingleCaseIsDisplayed();
```

Comment:
you can use the following assertion when you have added the two methods below into the Page.java class
homepage.assertElementIsDispalyed(homepage.createSingleCaseButton);

public void assertElementIsDisplayed(WebElementFacade element) {
        assertThat(isElementDisplayed(element), is(true));
    }

public void assertElementIsNotDisplayed(WebElementFacade element) {
        assertThat(isElementDisplayed(element), is(false));
    }